### PR TITLE
Add DynamoDB time series data gotcha

### PR DESCRIPTION
**Pull request by ustroetz**
First contribution here with two changes:

* Add DynamoDB time series data gotcha
* Remove of unnecessary white spaces

I had a look at the contribution guidelines, but please give me further feedback if it doesn't fit the format.

### CODE REVIEW

**Comment by ustroetz** [comment]
I resolved the merge conflicts.

File: README.md [hunk 1]
```diff
@@ -997,6 +997,7 @@ DynamoDB
...
 -	It is important to make sure that DynamoDB [resource limits](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Limits.html#limits-data-types) are compatible with your dataset and workload. For example, the maximum size value that can be added to a DynamoDB table is 400 KB (larger items can be stored in S3 and a URL stored in DynamoDB).
+-	Dealing with time series data in DynamoDB can be challenging. A Global Secondary Index together with down sampling timestamps can be a possible solution as explained [here](https://blogs.aws.amazon.com/bigdata/post/Tx3KPZDXIBJEQ4B/Scaling-Writes-on-Amazon-DynamoDB-Tables-with-Global-Secondary-Indexes).
```
Comment (jlevy) [inline README.md]: Boldface **time series data**

"Global Secondary Index" -> "global secondary index"

Add 🔸 at front

**Comment by ustroetz** [comment]
Thanks for the feedback. I incorporated the requested changes.

**Comment by forwidur** [comment]
LGTM. This is awesome!

Thanks a lot, @ustroetz !

**Pull request merged**